spec/features/pets/index_spec.rb

```diff
@@ -81,4 +81,13 @@
     expect(page).to have_content(pet_2.name)
```

Comment:
Is this passing? It seems like you'll need @ before pet_2 and pet_3

Reply:
This does pass (fyi, this test was all from the original files we forked - I didn't write any of this). If you look at line 72, `pet_2 = Pet.create(adoptable: true, age: 3, breed: "domestic pig", name: "Babe", shelter_id: shelter.id)` ,  pet_2 is created as a simple variable (no @) in the "it" block. Therefore, no @ sign needed when calling it inside of the same "it" block